tiatoolbox/annotation/storage.py

```diff
@@ -2180,11 +2171,17 @@ def _query(
                 The columns to select.
             geometry(tuple or Geometry):
                 The geometry being queried against.
-            select_callable(str):
-                The rows to select when a callable is given to `where`.
             callable_columns(str):
                 The columns to select when a callable is given to
                 `where`.
+            geometry_predicate(str):
```

Comment:
@John-P Please can you check this?

Reply:
What needs checking here? I am unclear on this one. The docstring for geometry_predicate looks correct except for a missing -s on defines

Reply:
I realised this may have been meanign to refer to min_area? I have added this.